firmware/tunerstudio/rusefi.input

```diff
@@ -3930,27 +3930,27 @@ dialog = tcuControls, "Transmission Settings"
 
 	help = fuelHelp, "Fuel Control"
 		text = "More about fuel control on the web"
-		webHelp = "https://rusefi.com/s/fuel"
+		webHelp = "https://wiki.fome.tech/r/fuel"
 
 	help = triggerHelp, "Trigger Settings"
 		text = "More about Trigger Settings on the web"
-		webHelp = "https://rusefi.com/s/trigger"
+		webHelp = "https://wiki.fome.tech/r/trigger"
 
 	help = etbHelp, "ETB Control"
 		text = "More about electronic throttle body on the web"
-		webHelp = "https://rusefi.com/s/etb"
+		webHelp = "https://wiki.fome.tech/r/etb"
 
 	help = vrThresholdHelp, "VR Sensor Threshold"
-		text = "This is about rusEFI VR Discrete decoder"
-		webHelp = "https://rusefi.com/"
+		text = "This is about FOME VR Discrete decoder"
+		webHelp = "https://wiki.fome.tech"
```

Comment:
missed one?

Reply:
Good catch, thanks!

Added in https://github.com/FOME-Tech/wiki/commit/d0b87f27e0093e6d8297a402718f89a6fb317efc